config/filter.go

```diff
@@ -20,6 +20,11 @@ type TypeFilterConfig interface {
 	CommonFilter(context.Context, logevent.LogEvent) logevent.LogEvent
 }
 
+// IsConfigured returns whether common configuration has been setup
+func (f *FilterConfig) IsConfigured() bool {
```

Comment:
Can you add unit test for this function?

Reply:
Sure. done